src/main.rs

```diff
@@ -194,7 +189,7 @@ pub fn init_logger() {
                 Naming::Timestamps,
                 Cleanup::KeepLogFiles(1),
             ),
-        false => Logger::with_env_or_str(format!("neovide = {}", verbosity)),
+        false => Logger::with_env_or_str(format!("neovide = {}", "trace")),
```

Comment:
Why did you remove the `verbosity` option and force the log level to `trace`? People in general don't want the console spammed with debug output when running neovide (not to mention that this has performance implications)!

Reply:
Does this spam to console? The logger is supposed to only write to file when the log argument is given. All other times the log calls are effectively noops

Maybe there is behavior I don't understand on some platforms?

Reply:
yes, it does...

Reply:
I think that's configurable; can't find out at first glance what the default is for flexi_logger

Reply:
https://docs.rs/flexi_logger/latest/flexi_logger/code_examples/index.html#choose-the-log-output-channel:

> By default, logs are written to stderr.

Reply:
Welp. I can fix this :)

Reply:
There's already a PR up: https://github.com/neovide/neovide/pull/1085

Reply:
I think the verbosity flag is unnecessary if we only log out when --log is present. That way the log always has all the information we need and never effects performance when we don't.

I think the goal originally was to always print errors to the console. Which I may still try to get going

Reply:
Makes sense; you can do that pretty easily with `flex_logger`.